hw/ip/tlul/data/tlul.prj.hjson

```diff
@@ -4,13 +4,23 @@
 
 {
     name:               "tlul",
-    version:            "0.5",
-    life_stage:         "L2",
-    design_stage:       "D3",
-    verification_stage: "V3",
-    notes:              '''the target for this entry is the autogenerated xbar_main within top_earlgrey.
-
-                        Signoff commit id:
-                        [f723f32](https://github.com/lowRISC/opentitan/tree/f723f32)
-                        ''',
+    revisions: [
+      {
+        version:            "0.5",
+        life_stage:         "L2",
+        design_stage:       "D3",
+        verification_stage: "V3",
+        commit_id:          "f723f327bae4c70a14409ab260ddcb7856be2ebd",
+        notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey."
+      }
+      {
+        version:            "1.0",
+        life_stage:         "L1",
+        design_stage:       "D1",
+        verification_stage: "V1",
+        commit_id:          "0adb90e886ee9d9a2e931ff789130d0229923e89",
```

Comment:
I cannot find the commit.
https://github.com/lowrisc/opentitan/tree/0adb90e886ee9d9a2e931ff789130d0229923e89

The commit ID should be visible before this PR is created.

Reply:
Updated, PTAL